src/app/home-page/home-page.component.scss

```diff
@@ -43,6 +43,14 @@ h2 {
     margin-bottom: 10px;
     padding-top: 30px;
   }
+
+  @media all and (min-width: 0px) and (max-width: 768px) {
+    padding: 0 10px 30px 10px;
+  }
+
+  @media all and (min-width: 769px) and (max-width: 1366px) {
+    padding: 0 30px;
+  }
```

Comment:
Did you review #5 and #6? If yes,

1. Any reason why we're not doing mobile-first here?
2. Where does 1366 viewport come from?